pkg/beacon/beacon.go

```diff
@@ -76,65 +78,58 @@ func Initialize(
 	node.ResumeSigningIfEligible(relayChain, signing)
 
 	relayChain.OnRelayEntryRequested(func(request *event.Request) {
-		currentEntryStartBlock, err := relayChain.CurrentRequestStartBlock()
-		if err != nil {
-			logger.Warningf("could not check current request start block")
-			return
-		}
+		onConfirmed := func() {
+			if node.IsInGroup(request.GroupPublicKey) {
```

Comment:
Should we check if member is in the group here or before calling `confirmCurrentRelayRequest`?

Reply:
I don't think so. If the node is not in a signing group it sill enters the signing in a courteous message relay mode. We need it until we update the client to the most recent libp2p with the fix @lukasz-zimnoch implemented for pubsub relays. We do not want the client to enter this mode for old relay entries. Also, no matter if the node is in the group or not, it starts monitoring the chain for relay entry timeouts. Similarly, we do not want the node to do that if the event received is not for the most recent relay request.